tests/cases/standalone/common/timestamp/timestamp.sql

```diff
@@ -1,3 +1,5 @@
+SELECT timestamp '    2017-07-23    13:10:11   ';
```

Comment:
How about also adding `SELECT ' 2017-07-23 13:10:11 '` to make sure the normal string literal is not affected